Deploy documentation

ai-qui opened the pull request:
Configure documentation deployement on readthedocs.io
cf. #9

adriengoeller reviewed (approved):
Issue already discussed for the html inclusion, and solved. Well done !

ai-qui commented:
Hello @adriengoeller, I've rebased this branch on the main branch; also I've made a little change to mkdocs configuration (more precisely, I've removed 2 useless lines in the configuration of a markdown extension).

adriengoeller reviewed (approved):
Ok for me !
